packages/fe-fpm-writer/test/unit/page/object.test.ts

```diff
@@ -131,6 +150,25 @@ describe('ObjectPage', () => {
             expect((fs.readJSON(join(target, 'webapp/manifest.json')) as any)?.['sap.ui5'].routing).toMatchSnapshot();
         });
 
+        test('simple nested navigation with contextPath', () => {
+            const target = join(testDir, 'with-nested-nav');
+            fs.write(join(target, 'webapp/manifest.json'), testAppManifest);
+            generate(
+                target,
+                {
+                    ...minimalInput,
+                    contextPath: '/RootEntity/OtherEntity',
```

Comment:
if the navigation is called to_OtherEntity the contextPath also needs to be /RootEntity/to_OtherEntity